sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentenceSentiment.java

```diff
@@ -14,25 +15,42 @@
     private final String text;
     private final SentimentConfidenceScores confidenceScores;
     private final TextSentiment sentiment;
+    private final IterableStream<MinedOpinion> minedOpinions;
 
     /**
      * Creates a {@link SentenceSentiment} model that describes the sentiment analysis of sentence.
-     *
-     * @param text The sentence text
+     * @param text The sentence text.
      * @param sentiment The sentiment label of the sentence.
      * @param confidenceScores The sentiment confidence score (Softmax score) between 0 and 1, for each sentiment label.
      *   Higher values signify higher confidence.
      */
     public SentenceSentiment(String text, TextSentiment sentiment, SentimentConfidenceScores confidenceScores) {
         this.text = text;
         this.sentiment = sentiment;
+        this.minedOpinions = null;
+        this.confidenceScores = confidenceScores;
+    }
+
+    /**
+     * Creates a {@link SentenceSentiment} model that describes the sentiment analysis of sentence.
+     * @param text The sentence text.
+     * @param sentiment The sentiment label of the sentence.
+     * @param minedOpinions The mined opinions of the sentence sentiment.
```

Comment:
should include this is only returned if you pass the opinion mining parameter to the `analyzeSentiment` cal